static/images/doubleleftarrow.svg

```diff
@@ -0,0 +1,43 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
These SVGs should be minified, ideally at build time. I recommend svgo for this, can easily shave off 30%.